source/options.css

```diff
@@ -1,17 +1,21 @@
 body {
 	min-width: 430px; /* Required to size the Browser Action popup */
 }
+input,
 textarea {
 	display: block;
 	width: 100%;
 	font-family: SFMono-Regular, Consolas, 'Liberation Mono', Menlo, Courier, monospace;
 	font-size: 11px;
 	line-height: 1.5;
+}
+textarea {
 	resize: vertical;
 	-moz-tab-size: 4 !important;
 	tab-size: 4 !important;
 }
 input[type='checkbox'] {
+	display: inline-block;
```

Comment:
You can undo this if the `input` selector above is `input:not([type=checkbox])`

Reply:
Also I'd add:

```css

:root {
	--github-green: #28a745;
	--github-red: #cb2431;
}
input:valid {
	color: var(--github-green);
	border: 1px solid;
}
input:invalid {
	color: var(--github-red);
	border: 1px solid;
}
```

Source: https://github.com/bfred-it/github-issue-link-status/commit/ee27e1d6e8e91978efad0698fb857117cfa626f2